.github/workflows/lint.yml

```diff
@@ -0,0 +1,110 @@
+# ESLint is a tool for identifying and reporting on patterns
```

Comment:
can you change the name back to `eslint` instead of `lint`? We have other linters that will be running

Reply:
unless we can run stylelint as well from this? I'm not sure it would be parallellized then 🤔

Reply:
yeah we can run stylelint from this as well,
thats why we cache it in setup, then the cache is reused for the lint jobs